sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageExtensions.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Core;
+using Microsoft.Azure.Amqp;
+using Microsoft.Azure.Amqp.Framing;
+
+namespace Azure.Messaging.ServiceBus.Amqp
+{
+    internal static class AmqpMessageExtensions
+    {
+        public static AmqpMessage ToAmqpMessage(this ServiceBusMessage message) =>
```

Comment:
It was in a separate file like this originally because there were multiple (underlying) implementations per body type. Also, if you move the 2 public methods from this class, you have to move everything out of it. The other class is already jam-packed with a bunch of different junk. For now, this class is only holding body conversion information. How about, I change the name to be related to Body conversion and not make them extension methods? Sound good?

Reply:
I'm good either way you decide.  As an industry, I feel that we sometimes seem to overvalue not repeating code at the cost of additional layers of abstraction.  I don't think there's a right or wrong answer here, it's author preference.   I was just raising the perspective for consideration.

Reply:
I think I'll keep it as is. There would likely be changes anyway once we decide on a way to expose the AMQP body types.